constraints/README.md

```diff
@@ -0,0 +1,68 @@
+# Accelerated table data quality with constraint enforcement
+
+This sample demonstrates how to use Spice to enforce constraints on locally accelerated data. This can be especially useful when you have a `refresh_mode: append` accelerated dataset and the data can be updated in the datasource.
+
+By specifying a `time_column` on the dataset with `refresh_mode: append` on the acceleration, Spice will automatically pull in all changes from the datasource that have occurred after the max timestamp in the accelerated dataset. This can present a problem if the datasource updates the data and it appears as a new row in the accelerated dataset, when it should have updated an existing row.
+
+This sample will have a local Postgres database with a table `users` and a Spice runtime that accelerates the data from the `users` table. A Spicepod will enforce a constraint that the `email` column must be unique. The Spicepod will also specify a `time_column` of `updated_at` to ensure that the Spice runtime only pulls in changes from the datasource that have occurred after the max `updated_at` timestamp in the accelerated dataset. A worker service will update the `users` table in the Postgres database with changes to the `users` table every 5 seconds. This will cause the Spice runtime to pull in the changes and enforce the constraint.
+
+Once you've verified that the constraints are being enforced, try modifying the Spicepod to remove the `on_conflict` clause and observe the behavior. An error should now be given by DuckDB that the constraint is being violated and the data update is rejected.
```

Comment:
nit. Put this and next line into `How to run` section, it could be easy to miss these changes.

Reply:
This is explaining how the sample works, not how to run it.